doc/modules/gaussian_process.rst

```diff
@@ -470,13 +471,13 @@ It is defined as:
 Kernel operators
 ----------------
 Kernel operators take one or two base kernels and combine them into a new
-kernel. The :class:`Sum` kernel takes two kernels :math:`k1` and :math:`k2`
-and combines them via :math:`k_{sum}(X, Y) = k1(X, Y) + k2(X, Y)`.
-The  :class:`Product` kernel takes two kernels :math:`k1` and :math:`k2`
-and combines them via :math:`k_{product}(X, Y) = k1(X, Y) * k2(X, Y)`.
+kernel. The :class:`Sum` kernel takes two kernels :math:`k_1` and :math:`k_2`
+and combines them via :math:`k_{sum}(X, Y) = k_1(X, Y) + k_2(X, Y)`.
+The  :class:`Product` kernel takes two kernels :math:`k_1` and :math:`k_2`
+and combines them via :math:`k_{product}(X, Y) = k_1(X, Y) * k_2(X, Y)`.
 The :class:`Exponentiation` kernel takes one base kernel and a scalar parameter
-:math:`exponent` and combines them via
-:math:`k_{exp}(X, Y) = k(X, Y)^\text{exponent}`.
+:math:`p` and combines them via
+:math:`k_{exp}(X, Y) = k(X, Y)^p`.
```

Comment:
Maybe add a (technical but useful) detail e.g.

Note that magic methods `__add__`, `__mul` and `__pow__` are overridden on the Kernel objects, so one can use e.g. `RBF() + RBF()` as a shortcut for `Sum(RBF(), RBF())`